docs/crio.conf.5.md

```diff
@@ -108,6 +108,10 @@ Example:
 **default_mounts**=[]
   List of mount points, in the form host:container, to be mounted in every container
 
+**read_only**==*true*|*false*
+  Run every container in read-only mode. Automatically mount tmpfs on /run, /tmp and /var/tmp.  Setup
```

Comment:
I think the `Automatically mount …` bit needs to go in `crio.8.md` as well.  We could probably DRY that up by having one man page reference the other and explain the setting-mapping algorithm, but the current approach seems to be landing the full docs for each setting in both files.

We may also want to link [here][1] for “read-only mode”, in case folks wonder about what that means.  And maybe also point out that this only matters for new mount namespaces (and not for `exec`, etc.).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#root

Reply:
@umohnani8 Copy the description there also please.

Reply:
fixed.